var/spack/repos/builtin/packages/xsdk-examples/package.py

```diff
@@ -92,3 +92,8 @@ def cmake_args(self):
             args.append('-DZLIB_LIBRARY_DIR=%s' % spec['zlib'].prefix.lib)
 
         return args
+
+    def check(self):
+        if self.run_tests:
```

Comment:
You can remove this line, `check` is only run when this is true

Reply:
Ok done.

The prior format was a copy/paste from current usage in other packages.